src/drivers/distance_sensor/lightware_laser_i2c/parameters.c

```diff
@@ -48,3 +48,15 @@
  * @value 7 SF/LW30/d
  */
 PARAM_DEFINE_INT32(SENS_EN_SF1XX, 0);
+
+/**
+ * Lightware SF1xx/SF20/LW20 Operation Mode
+ *
+ * @value 0 Disabled
+ * @value 1 Always On
+ * @value 2 Auto: enabled/disabled via front/back transitions.
```

Comment:
```suggestion
 * @value 2 Disabled during VTOL fast forward flight
```
Just a suggestion. If the option text is too long it doesn't show nicely in the UI so maybe we make it even shorter and add more details in the parameter description.